nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp

```diff
@@ -325,7 +330,6 @@ geometry_msgs::msg::TwistStamped RegulatedPurePursuitController::computeVelocity
     rotateToHeading(linear_vel, angular_vel, angle_to_heading, speed);
   } else {
     applyConstraints(
```

Comment:
This call does not contain the `transformed_plan`? Don't believe this compiles

Reply:
Yes, I haven't fully fleshed this out yet. I will tie up all the loose ends when I get time.